examples/landing-zone-v2/tier3/dns/address.yaml

```diff
@@ -17,7 +17,6 @@ apiVersion: compute.cnrm.cloud.google.com/v1beta1
 kind: ComputeAddress
 metadata:
   name: workload-name-external-ip # kpt-set: ${workload-name}-external-ip
-  namespace: project-id-tier4 # kpt-set: ${project-id}-tier4
```

Comment:
Should we leave the namespace but make it generic like `config-control` or is this example assuming they are just deploying into the namepace of their kubecontext?

Reply:
@lucstjean-ssc here